lib/discourse_api/api/uploads.rb

```diff
@@ -8,6 +8,14 @@ def upload_post_image(args)
                   .to_h
         post('/uploads', args)
       end
+
+      def upload_file(args)
```

Comment:
I don't think we need to have another method here. I think we can use the existing one, do you know if it matters if we use type: 'composer' vs 'upload'? If it matters we could make it an optional argument.